### PR TITLE
Enforce using https during setup when accessing Github API

### PR DESCRIPTION
**Pull request by jpawlowski**

### CODE REVIEW

**Review by AlCalzone — COMMENTED** [review]
Something messed up the formatting in the .js files.

**Comment by Apollon77** [comment]
Yes, please fix the formatting ...

**Comment by jpawlowski** [comment]
To me it seems this repository comes with some presets for the Prettier extension in VScode ([.prettierignore](https://github.com/ioBroker/ioBroker.js-controller/blob/master/.prettierignore) and [.prettierrc.json](https://github.com/ioBroker/ioBroker.js-controller/blob/master/.prettierrc.json)). There might be incompatibilities with the settings you put in [.eslintrc.js](https://github.com/ioBroker/ioBroker.js-controller/blob/master/.eslintrc.js), resulting in this failure.

If you want me to fix the formatting, I suggest that this gets sorted out in the first place? I didn't change the formatting intentionally but simply followed what is configured for this repo.

**Comment by foxriver76** [comment]
>  There might be incompatibilities with the settings you put in [.eslintrc.js](https://github.com/ioBroker/ioBroker.js-controller/blob/master/.eslintrc.js), resulting in this failure.

There should be no, @AlCalzone is working with VSCode too and it is all fine. Have you actually installed the Prettier extension for the project? Else default configurations of your editor might be applied.

File: schemas/io-package.json [hunk 1]
```diff
@@ -1,1275 +1,1270 @@
...
-                "additionalProperties": false,
-                "required": [
-                  "name"
-                ],
-                "properties": {
-                  "name": {
-                    "type": "string"
-                  },
-                  "email": {
+    "$schema": "https://json-schema.org/draft-07/schema",
```
Comment (Apollon77) [inline schemas/io-package.json:2]: is that really also relevant for schema urls??

Comment (AlCalzone) [reply schemas/io-package.json:2]: Doesn't hurt IMO, so why not?

Comment (Apollon77) [reply schemas/io-package.json:2]: kk, so we "just" need to fix the intents or take over manually ... for the rest

**Pull request closed without merging**